tensorflow/core/kernels/adjust_hue_op_test.cc

```diff
@@ -0,0 +1,56 @@
+/* Copyright 2015 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include <cmath>
+#include <vector>
+#include "tensorflow/core/framework/allocator.h"
+#include "tensorflow/core/framework/fake_input.h"
+#include "tensorflow/core/framework/graph.pb.h"
+#include "tensorflow/core/framework/node_def_builder.h"
+#include "tensorflow/core/framework/op_kernel.h"
+#include "tensorflow/core/framework/tensor.h"
+#include "tensorflow/core/framework/tensor_testutil.h"
+#include "tensorflow/core/framework/types.h"
+#include "tensorflow/core/framework/types.pb.h"
+#include "tensorflow/core/kernels/ops_testutil.h"
+#include "tensorflow/core/kernels/ops_util.h"
+#include "tensorflow/core/lib/core/status_test_util.h"
+#include "tensorflow/core/platform/test.h"
+
+namespace tensorflow {
+
+class AdjustHueOpTest : public OpsTestBase {
+};
+
+TEST_F(AdjustHueOpTest, Simple_1113) {
+
+  TF_EXPECT_OK(NodeDefBuilder("adjust_hue_op", "AdjustHue")
+      .Input(FakeInput(DT_FLOAT))
+      .Input(FakeInput(DT_FLOAT))
+      .Finalize(node_def()));
+
+  TF_EXPECT_OK(InitOp());
+
+  AddInputFromArray<float>(TensorShape({1, 1, 1, 3}), {127, 137, 225});
```

Comment:
There are a lof of python tests under

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/image_ops_test.py

Does this test do something special? 

Please make sure all the tests will use the new GPU kernel, and pass after this change.